letsencrypt/client/client.py

```diff
@@ -336,7 +336,8 @@ def init_csr(privkey, names, cert_dir):
     csr_pem, csr_der = crypto_util.make_csr(privkey.pem, names)
 
     # Save CSR
-    le_util.make_or_verify_dir(cert_dir, 0o755)
+    if not os.environ.get('DOCKER_RUN'):
```

Comment:
i don't really understand why this is necessary :( If you could please provide some comment inline, in the source code, that would be awesome